modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc

```diff
@@ -115,7 +115,7 @@ To import to a running workspace using the *Git: Clone* command:
 +
 image::workspaces/{project-context}-welcome.png[Welcome screen]
 
-. Open the command palette using `F1` or `CTRL-SHIFT-P`, or from the link in the Welcome screen.
+. Open the command palette using `F1` or `CTRL-SHIFT-P` or using the link displayed on the Welcome screen.
```

Comment:
```suggestion
. Open the command palette by using `F1`, `CTRL-SHIFT-P`, or the link displayed on the Welcome screen.
```
To avoid too many "or"s :D